dolphinscheduler-scheduler-plugin/dolphinscheduler-scheduler-quartz/src/main/java/org/apache/dolphinscheduler/scheduler/quartz/QuartzScheduler.java

```diff
@@ -111,7 +111,7 @@ public void insertOrUpdateScheduleTask(int projectId, Schedule schedule) throws
                     .endAt(endDate)
                     .withSchedule(
                             cronSchedule(cronExpression)
-                                    .withMisfireHandlingInstructionDoNothing()
+                                    .withMisfireHandlingInstructionFireAndProceed()
```

Comment:
Good catch, we miss this.

This parameter works with `org.quartz.jobStore.misfireThreshold`, can we make this configurable? Maybe some user don't want to open this.

Reply:
+1